server/eventlog.go

```diff
@@ -127,9 +157,22 @@ func getVerifiedCosState(coscel cel.CEL) (*pb.AttestedCosState, error) {
 
 	seenSeparator := false
 	for _, record := range coscel.Records {
-		// COS State only comes from the CosEventPCR
-		if record.PCR != cel.CosEventPCR {
-			return nil, fmt.Errorf("found unexpected PCR %d in CEL log", record.PCR)
+		// if a record is not using under trustingRegisterType, skip the record
```

Comment:
Throw an error for now. We can always support two register types, but this makes an event log easier to reason about.

Reply:
done